roles/splunk/defaults/main.yml

```diff
@@ -69,6 +70,10 @@ splunk_shc_target_group: shc
 splunk_shc_deployer: "{{ groups['shdeployer'] | first }}" # If you manage multiple SHCs, configure the var value in group_vars
 splunk_shc_uri_list: "{% for h in groups[splunk_shc_target_group] %}https://{{ hostvars[h].ansible_fqdn }}:{{ splunkd_port }}{% if not loop.last %},{% endif %}{% endfor %}" # If you manage multiple SHCs, configure the var value in group_vars
 start_splunk_handler_fired: false # Do not change; used to prevent unnecessary splunk restarts
+splunk_enable_kvstore: true
+splunk_kvstore_storage: undefined # Can be defined here or at the group_vars level - accepted values: "wiredTiger" or "undefined", which leaves as default
+splunk_kvstore_version: undefined # Can be defined here or at the group_vars level - accepted values: 4.2 or "undefined", which leaves as default1
```

Comment:
I don't see this variable used either

Reply:
It's used here: https://github.com/splunk/ansible-role-for-splunk/pull/177/commits/9f689aa08ddf78dd87a56cde663ff497a42dc920#diff-0977dd8c4cb1bc6903196d2c0c0efd06d11183fbdb1dccf3620ffff234438804R28

Reply:
Oh I see, the `splunk_kvstore_version` is unused - I'll add it to the conditionals for the bottom of the upgrade procedure